client/my-sites/email/email-management/email-home.tsx

```diff
@@ -15,6 +15,7 @@ import EmailListActive from 'calypso/my-sites/email/email-management/home/email-
 import EmailListInactive from 'calypso/my-sites/email/email-management/home/email-list-inactive';
 import EmailNoDomain from 'calypso/my-sites/email/email-management/home/email-no-domain';
 import EmailPlan from 'calypso/my-sites/email/email-management/home/email-plan';
+import { IntervalLength } from 'calypso/my-sites/email/email-providers-comparison/interval-length';
```

Comment:
Shouldn't this be a type import?
```suggestion
import type { IntervalLength } from 'calypso/my-sites/email/email-providers-comparison/interval-length';
```

Reply:
Nope, because that's an enumeration and not a type 🙂.

Reply:
Fair enough!